lib/pandadoc/documents/documents.ex

```diff
@@ -136,7 +140,8 @@ defmodule Pandadoc.Documents do
 
   https://developers.pandadoc.com/reference/change-document-status-manually
   """
-  @spec change_document_status(Pandadoc.client(), String.t(), valid_document_statuses()) :: Pandadoc.result()
+  @spec change_document_status(Pandadoc.client(), String.t(), valid_document_statuses()) ::
+          Pandadoc.result()
```

Comment:
@seanhuggins1 is this from your formatter?

Reply:
Yeah, it must have not been formatted before? We don't have pre-commit formatting in this repo